src/methods/scgpt_zero_shot/config.vsh.yaml

```diff
@@ -0,0 +1,39 @@
+__merge__: ../../api/comp_method.yaml
+
+name: scgpt_zero_shot
+label: scGPT zero-shot
+summary: "Reference mapping using cell embedding by pretrained scGPT model."
+description: |
+  scGPT is a foundation model for single-cell biology based on a generative pre-trained transformer and trained on a repository of over 33 million cells. Following the zero-shot approach, a pre-trained scGPT model is used to embed cells and map unlabelled cells in a query set to the reference dataset with provided annotations. 
```

Comment:
Should we mention this uses nearest neighbours rather then the scGPT classifier?

Reply:
Yes, I'll rewrite the description.